package/upgrade/upgrade_node.sh

```diff
@@ -660,6 +675,10 @@ stages:
 EOF
   fi
 
+  # SLE Micro 5.5 uses /usr/lib/ssh/sftp-server
+  # SL Micro 6.1 uses /usr/libexec/ssh/sftp-server
+  sed -i 's%/usr/lib/ssh/sftp-server%/usr/libexec/ssh/sftp-server%' ${HOST_DIR}/etc/ssh/sshd_config.d/sftp.conf
```

Comment:
unless SFTP is enabled in harvester config the directory and sftp config file will not exist.
we should gate this `sed` in a check which verifies that file exists